samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -105,62 +162,174 @@ static struct hid_ops ops = {
 	.get_report = get_report_cb,
 };
 
-static int def_val;
-static volatile int status;
+static u32_t def_val[4];
```

Comment:
It may be correct, but not nice to change the type of the variable in this commit.

Reply:
Should I make a separate one?

Reply:
I think yes, would be more accurate :-)

Reply:
Done :)